aap_compose_dev.yaml

```diff
@@ -200,12 +227,91 @@ services:
       - ".:/src/galaxy_ng"
     environment:
       <<: *common-env
+    user: root
     command: |
       bash -c "
         while [[ ! -f /var/lib/pulp/.migrated ]]; do
           echo 'Waiting for migrations ...';
           sleep 2;
-        done && exec pulpcore-worker;
+        done;
+        while [[ ! -f /etc/pulp/certs/ansible-sign.key ]]; do
+          echo 'Waiting for signing key';
+          sleep 2;
+        done;
+
+        echo '#> STEP: Import GPG Keys for content signing tasks';
+        gpgconf --kill gpg-agent && gpg --batch --no-default-keyring --import /etc/pulp/certs/ansible-sign.key;
+        (echo 5; echo y; echo save) | gpg --command-fd 0 --no-tty --no-greeting -q --edit-key 'FB8B3F2D24BCAF7EFDF793A9F37575C52D4F16F3' trust;
+        gpg --list-secret-keys;
+        echo 'DONE! <#';
+
+        exec pulpcore-worker;
+      "
+
+  manager:
```

Comment:
Added this new container because commands such as `add-signing-service` and `set-repo-keyring` and `task-scheduler` relies on a running worker, so cannot execute on worker container before worker process is running, it would be a dead-lock to run anything that relies on tasks inside worker container.